lib/tasks/dev.rake

```diff
@@ -6,7 +6,24 @@ if Rails.env.development? || Rails.env.test?
     task prime: "db:setup" do
       include FactoryBot::Syntax::Methods
 
-      # create(:user, email: "user@example.com", password: "password")
+      create(:budget, user: User.create(
+        name: "admin",
+        email: "admin@example.com",
+        password: "secret",
+        admin: true
+      ))
+
+      9.times do
+        create(:budget, user: create(:user))
+      end
+
+      User.all.each do |user|
+        9.times do
+          create(:expense, budget: user.budget)
```

Comment:
This was a bit confusing at first, cause I thought you are recreating a budget for each user.
A suggestion to build this into the first `9.times` block. For example:
```
9.times do
  budget = create(:budget)
  create(:expense, budget: budget)
end
```

or
```
9.times { create(:budget, expenses: [create(:expense)] }
```

or
```
9.times { create(:budjet, :with_expenses) }
```
And defined the `:with_expenses` in the factory as an `after_create` block

Reply:
Done in sha: 7c2f098